spec/rails_helper.rb

```diff
@@ -9,9 +9,25 @@
 
 require 'capybara-screenshot/rspec'
 
-# Poltergeist with PhantomJS
-require 'capybara/poltergeist'
-Capybara.javascript_driver = :poltergeist
+# Headless Chrome
+require 'selenium/webdriver'
+
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+
+Capybara.register_driver :selenium do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu window-size=1366,768) }
+  )
+
+  Capybara::Selenium::Driver.new app,
+    browser: :chrome,
+    desired_capabilities: capabilities
+end
+
+Capybara.javascript_driver = :selenium
```

Comment:
Not sure what's currently executing with this, but did you want to use Chrome headless as for JS as well? `
Capybara.javascript_driver = :headless_chrome`